./gradlew publishToMavenLocal failed on macOS

### How frequently does the bug occur?

Always

### Description

`./gradlew publishToMavenLocal` failed on macOS.

### Stacktrace & log output

```shell
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':packages:cinterop:iosArm64SourcesJar' (type 'Jar').
  - Gradle detected a problem with the following location: '/Users/tinoue/git/private/krdb/packages/cinterop
/build/generated/source/version'.
    
    Reason: Task ':packages:cinterop:iosArm64SourcesJar' uses this output of task ':packages:cinterop:genera
teSdkVersionConstant' without declaring an explicit or implicit dependency. This can lead to incorrect resul
ts being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':packages:cinterop:generateSdkVersionConstant' as an input of ':packages:cinterop:ios
Arm64SourcesJar'.
      2. Declare an explicit dependency on ':packages:cinterop:generateSdkVersionConstant' from ':packages:c
interop:iosArm64SourcesJar' using Task#dependsOn.
      3. Declare an explicit dependency on ':packages:cinterop:generateSdkVersionConstant' from ':packages:c
interop:iosArm64SourcesJar' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.12/userguide/validation_problems.html#im
plicit_dependency in the Gradle documentation.
```

### Can you reproduce the bug?

Always

### Reproduction Steps

./graldew claen 
./gradlew jvmTest 
./gradlew publishToMavenLocal

### Version

3.2.6

### What Atlas App Services are you using?

Local Database only

### Are you using encryption?

No

### Platform OS and version(s)

macOS 15.1

### Build environment

Android Studio version: Meerkat
Android Build Tools version: 36 rev 2
Gradle version: 8.12

--- a/packages/cinterop/build.gradle.kts
+++ b/packages/cinterop/build.gradle.kts
@@ -877,4 +877,19 @@ afterEvaluate {
     tasks.named("jvmSourcesJar") {
         dependsOn("generateSdkVersionConstant")
     }
+    tasks.named("iosArm64SourcesJar") {
+        dependsOn("generateSdkVersionConstant")
+    }
+    tasks.named("iosSimulatorArm64SourcesJar") {
+        dependsOn("generateSdkVersionConstant")
+    }
+    tasks.named("iosX64SourcesJar") {
+        dependsOn("generateSdkVersionConstant")
+    }
+    tasks.named("macosArm64SourcesJar") {
+        dependsOn("generateSdkVersionConstant")
+    }
+    tasks.named("macosX64SourcesJar") {
+        dependsOn("generateSdkVersionConstant")
+    }
 }

